website/docs/docs/login.md

```diff
@@ -5,9 +5,19 @@ title: Login
 
 This section describes certain issues you might face while logging into PactFlow and explains how to resolve them.
 
-## Unable to login into PactFlow user interface
+## *"You are not authorized to access this Pactflow account"*
 
-When you sign up for PactFlow, you will be emailed the login details with a temporary password. This password is valid for 7 days. If you need to reset the password, you can reset the clock on this expiry by simply re-inviting the user. Alternatively, [contact us (https://support.smartbear.com/pactflow/message/) and we will reset it for you.
```

Comment:
When you sign up for PactFlow, you will get an email comprising the login details with a temporary password.